web/views/pages/index.ejs

```diff
@@ -9,10 +9,11 @@
         <meta property="og:type" content="website" />
         <!-- It's import to keep the old og:url, else, likes will not accumulate properly -->
         <meta property="og:url" content="http://electricitymap.tmrow.co" />
+
         <meta property="og:locale" content="<%= FBLocale %>" />
-        <% for(var i=0; i<supportedFBLocales.length; i++) {%>
+<% for(var i=0; i<supportedFBLocales.length; i++) { -%>
```

Comment:
Without the `-` at the end, the loop adds newlines which makes the HTML cumbersome to inspect.
I also changed the indentation to produce proper well-indented HTML